.github/pull_request_template.md

```diff
@@ -3,7 +3,7 @@
 _What is this about?_
 
 ## PR Checklist
-* [ ] Applies to #xxx
+* [ ] Closes #xxx
```

Comment:
But sometimes it doesn't close the issue and so i would write:
```Markdown
 * [ ] Closes #xxx 
 <!-- Or:
 * [ ] Applies to #xxx 
 -->
```

Reply:
Well it does not work with `Applies To`.

Reply:
I know. But this would'nt be used often

Reply:
Linked issues can be detected by msftbot to add `In-Progress` label and things like that.

Reply:
Yes but sometimes a PR don't close a issue. And if thats so you can take the optional Applies to instead of Closes. But I think that wouldn't be used by many people so it isn't a problem